### ISSUE TITLE
✍️ Write style guide

### ISSUE THREAD

**Issue by tatianamac** [tatianamac/selfdefined#208]
## Overview
The site is getting to a point where it could use a house style guide in written form. As I've edited most of the English entries, I've been able to catch most things.

## Considerations
- Use en_GB for any English entries.
- Write in active voice.
- Adhere to [TBD] style guide for spelling.
- Serve as a guidepost and starting point; no entry is written as permanent and can and should evolve to new information.

## Technical Considerations
- Would be footer-level content, not connected to future main/top navigation
- Should likely live within or at same level as documentation

**Comment by kittensyzygy** [comment]
This might be related. I've opened an issue about creating a formatting guide.
https://github.com/selfdefined/web-app/issues/339

**Comment by tatianamac** [comment]
@kittensyzygy Yes, definitely related! We could consider that almost a "subtask" of this one and adjust this description accordingly, as it's primarily focused on the markdown component of this.
